src/config/trip.config.ts

```diff
@@ -0,0 +1,9 @@
+import { config } from 'dotenv';
+config();
```

Comment:
is it needed here?

Reply:
using this way "config()" is necessary, for the second way it can be omitted